usrp_drivers/n200/set_affinity.py

```diff
@@ -0,0 +1,80 @@
+import subprocess as sp
+import os
+import itertools as it
+import sys
+import
+
+sys.path.append(os.environ["BOREALISPATH"])
+
+if __debug__:
+    sys.path.append(os.environ["BOREALISPATH"] + '/build/debug/utils/protobuf')  # TODO need to get this from scons environment, 'release' may be 'debug'
+else:
+    sys.path.append(os.environ["BOREALISPATH"] + '/build/release/utils/protobuf')
+
+sys.path.append(os.environ["BOREALISPATH"] + '/utils/zmq_borealis_helpers')
+import socket_operations as so
+
+
+def get_tids(process_name):
+    p1 = sp.Popen(['bash','/root/get_tids.sh', process_name], stdout=sp.PIPE)
+    #p2 = sp.Popen(['awk', "'{printf '%s\n', $1}'"], stdin=p1.stdout, stdout=sp.PIPE)
+
+
+    stdout = p1.communicate()
+    p1.stdout.close()
+    #print(cpus[0].splitlines())
+
+    tids = []
+
+    for line in stdout[0].splitlines()[2:]:
+    	split_line = line.split()
+
+    	tid = split_line[1]
+    	tids.append(tid)
```

Comment:
this might still have dashes and the word "TID" in it, you'll want to check the output for this